staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -50,6 +50,27 @@ type EventSink interface {
 	Patch(oldEvent *v1.Event, data []byte) (*v1.Event, error)
 }
 
+// CorrelatorOptions allows you to change the default of the EventSourceObjectSpamFilter
+// and EventAggregator in EventCorrelator
```

Comment:
describe zero-value behavior for all options (and consider making it easy to get the default behavior you would get from NewEventCorrelator by leaving all options fields empty (default zero-values in NewEventCorrelatorWithOptions)

Reply:
Done.